<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1589473435990' stop='1589473532869'>
    <name>Connection</name>
    <title>Connection</title>
    <test-cases>
        <test-case start='1589473435991' status='failed' stop='1589473494291'>
            <name>In-place with Free plan</name>
            <title>In-place with Free plan</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1589473456189' status='passed' stop='1589473493325'>
                    <name>Can start in-place connection</name>
                    <title>Can start in-place connection</title>
                    <attachments/>
                    <steps/>
                </step>
                <step start='1589473493325' status='passed' stop='1589473494250'>
                    <name>Can assert that site is connected</name>
                    <title>Can assert that site is connected</title>
                    <attachments/>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>TypeError: Cannot read property 'currentStep' of undefined</message>
                <stack-trace>TypeError: Cannot read property 'currentStep' of undefined
                    at Allure.Object.&lt;anonymous>.Allure.addAttachment (/home/travis/build/Automattic/jetpack/node_modules/allure-js-commons/index.js:86:45)
                    at Reporter.addAttachment (/home/travis/build/Automattic/jetpack/node_modules/jest-allure/dist/Reporter.js:60:21)
                    at addAttachment (/home/travis/build/Automattic/jetpack/tests/e2e/lib/setup-env.js:26:12)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.(anonymous function) [as next] (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1589473494291' status='failed' stop='1589473526694'>
            <name>In-place upgrading a plan from personal to premium</name>
            <title>In-place upgrading a plan from personal to premium</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1589473501032' status='undefined'>
                    <name>Can set a sandbox cookie</name>
                    <title>Can set a sandbox cookie</title>
                    <attachments>
                        <attachment title='Test failed: Connection :: In-place upgrading a plan from personal to premium' source='efff4b67-d70d-41d2-8454-80a5cf22354b-attachment.png' type='image/png' size='23773'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>TypeError: Cannot read property 'currentStep' of undefined</message>
                <stack-trace>TypeError: Cannot read property 'currentStep' of undefined
                    at Allure.Object.&lt;anonymous>.Allure.addAttachment (/home/travis/build/Automattic/jetpack/node_modules/allure-js-commons/index.js:86:45)
                    at Reporter.addAttachment (/home/travis/build/Automattic/jetpack/node_modules/jest-allure/dist/Reporter.js:60:21)
                    at addAttachment (/home/travis/build/Automattic/jetpack/tests/e2e/lib/setup-env.js:26:12)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.(anonymous function) [as next] (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>